### ISSUE TITLE
Is this still in active development? (Question)

### ISSUE THREAD

**Issue by alexanderglueck** [tomschlick/townhouse#5]
Hey, 
just wanted to know whether you are still actively developing this solution?

**Comment by tomschlick** [comment]
Yes it is! Sorry for the extended delay! Working on this again this weekend and hoping to get an initial release out by the end of it.

**Comment by kaytotes** [comment]
Just watched your Laracon US 2017 talk and very interested. Will be keeping my eyes peeled.

**Comment by visualcookie** [comment]
Still interested, as `hyn/multi-tenant` seems kind of complicated. The docs are not that well documentating stuff.

**Comment by dahabit** [comment]
Hi Tom , Thanks for your effort , I'm still waiting since your last Laracon :)

**Comment by coreycoburn** [comment]
@visualcookie The creator of `hyn/multi-tenant` is creating a successor to that package, [https://github.com/tenancy/tenancy](https://github.com/tenancy/tenancy). Maybe that package will be less complicated.

However, @tomschlick I as well am still interested in townhouse after watching your Laracon talk and also listening to Full Stack Radio.

**Comment by OwenMelbz** [comment]
@tomschlick Has this currently got everything you mentioned in https://multitenantlaravel.com/ or are there still loose ends that need tying up?

We're starting a new multi db application, after using Landlord which was very straight forward, hoping to find something as intuitive for multi db.

Would you say what you have here now is useable in production? or would you be a little wary?

Thanks :)

**Comment by johans** [comment]
Enjoyed your talk @tomschlick and have the same question as @OwenMelbz. 

Is anyone using this currently?
